services/vrf/vrf_test.go

```diff
@@ -0,0 +1,122 @@
+package vrf
+
+import (
+	"crypto/rsa"
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func panicErr(err error) {
```

Comment:
[require.NoError](https://github.com/smartcontractkit/chainlink/blob/master/cmd/remote_client_test.go#L372) probably gives you a cleaner failure output, and the rest of the test suite will continue running if one assertion fails. Panic exits the whole test suite.